### DISCUSSION TITLE
how to get started?

### DISCUSSION THREAD

**Discussion by dulhaver** [opentofu/opentofu#1261] [Q&A] [answered]
I am **without any pre-existing experience of Terraform** and am looking for a place to start learning how to use opentofu.

So should I just look into Terraform tutorials to get going? I guess OpenToFu should still be pretty close to Terraform.

**Comment by ghost** [comment, marked as answer]
Hi @dulhaver at this point, Terraform tutorials will work equally well with OpenTofu.

**Reply by dulhaver** [reply]
ok, thx

**Reply by JoseAPerezB** [reply]
I am also looking to level up so that I can support OpenTofu.
I will be looking into this [tutorial](https://www.youtube.com/watch?v=7xngnjfIlK4). As a newbie, any better resources with free tutorials would be greatly appreciated.

**Reply by ghost** [reply]
Thank you for your feedback @JoseAPerezB , a set of guides is definitely on our todo list.

**Comment by JoseAPerezB** [comment]
I have time. If you need help, I can help the peeps working on the guides.
With the right guidance I could accelerate the development and publishing
of the guides.

**Reply by edmundf32** [reply]
@JoseAPerezB @abstractionfactory 

Has there been any update on this?. I would be interested in getting involved with a 'zero to Tofu' guide.

**Reply by ghost** [reply]
Hey @edmundf32 some, but not a lot. Can you join the #guides channel on the [OpenTofu Slack](https://opentofu.org/slack/)?

**Reply by edmundf32** [reply]
@abstractionfactory 

First thing I tried, getting an error. I don't suppose any of you are around in config management camp in Gent?

Cheers
![Screenshot_20250204-135633](https://github.com/user-attachments/assets/db7ac675-4822-4347-9347-a0d2d7bb74b8)

**Reply by ghost** [reply]
Hey @edmundf32 apologies, we'll fix that in a bit, until then please use [this link](https://join.slack.com/t/opentofucommunity/shared_invite/zt-2z21gw7qp-8PD1VG58ckZinXOjsYGU3A).

**Reply by edmundf32** [reply]
@abstractionfactory no bother, hey computers will computer. In now, cheers for that
